### PR TITLE
Parallel stats

### PR DESCRIPTION
**Pull request by gregtatum**
This adds the distribution stats to the parallel datasets as well. I initially only did the mono datasets to make the first implementation simpler. Here you will find the graphs are named after the datasets.

### CODE REVIEW

File: tests/fixtures/__init__.py [hunk 1]
```diff
@@ -161,23 +161,24 @@ def run_task(
             for part in command_parts
         ]
 
-        if extra_args:
```
Comment (gregtatum) [inline tests/fixtures/__init__.py:164]: These changes were required to support `&&` in the bash commands for the `run_task` abstraction.

**Comment by gregtatum** [comment]
Ah, I forgot to add this as a CI task.

**Comment by gregtatum** [comment]
Ok, it's working now, here as an example task: [analyze-corpus-opus-ada83_v1-ru-en](https://firefox-ci-tc.services.mozilla.com/tasks/FaBfDMq3S5SGbafX1GN3IA)

File: taskcluster/kinds/analyze-corpus/kind.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        run:
+            using: run-task
+            command:
+                - bash
+                - -c
+                - >-
+                    pip3 install --upgrade pip setuptools &&
+                    pip3 install -r $VCS_PATH/pipeline/data/requirements/analyze.txt
+                    &&
+                    python3 $VCS_PATH/pipeline/data/analyze.py
```
Comment (eu9ene) [inline taskcluster/kinds/analyze-corpus/kind.yml:64]: Maybe it works for now but I assume we'll want to do some analysis based on parallel sentences, like the ratio between the number of words etc. We might want to tune filters based on those per language. In this case, the analyze script would need to accept either two files as input.

**Review by eu9ene — APPROVED** [review]
Approving with an optional suggestion to accept two files as input for the script to make it more future-proof. This would also make the code related to handling ampersands go away.

**Review by bhearsum — APPROVED** [review]
Looks good from a Taskcluster standpoint.

**Review by jcristau — APPROVED** [review]

**Pull request merged**